PennMobile/Home/Fitness/FitnessRoomRow.swift

```diff
@@ -0,0 +1,220 @@
+//
+//  FitnessRoomRow.swift
+//  PennMobile
+//
+//  Created by Jordan H on 4/8/23.
+//  Copyright © 2023 PennLabs. All rights reserved.
+//
+
+import SwiftUI
+import Kingfisher
+import PennMobileShared
+
+struct FitnessRoomRow: View {
+    let room: FitnessRoom
+    @State var isExpanded = false
+    let meterSize: CGFloat = 80
+    let meterLineWidth: CGFloat = 6
+    var weekdayIndex: Int = (Calendar.current.component(.weekday, from: Date()) + 5) % 7
+    var hours: (Date, Date) {
+        let calendar = Calendar.current
+        let currentDate = Date()
+        
+        let timeFormatter = DateFormatter()
+        timeFormatter.dateFormat = "HH:mm:ss"
+        
+        let openTime = timeFormatter.date(from: room.open[weekdayIndex])!
+        let closeTime = timeFormatter.date(from: room.close[weekdayIndex])!
+
+        let openDate = calendar.date(bySettingHour: openTime.hour, minute: openTime.minutes, second: 0, of: currentDate)!
+        let closeDate = calendar.date(bySettingHour: closeTime.hour, minute: closeTime.minutes, second: 0, of: currentDate)!
+        
+        return (openDate, closeDate)
+    }
+    var isOpen: Bool {
+        let date = Date()
+        return hours.0 < date && date < hours.1
+    }
+
+    var body: some View {
+        VStack {
+            HStack(spacing: 13) {
+                KFImage(room.image_url)
+                    .resizable()
+                    .aspectRatio(contentMode: .fill)
+                    .frame(width: 100, height: 64)
+                    .background(Color.grey1)
+                    .clipShape(RoundedRectangle(cornerRadius: 7))
+                
+                VStack(alignment: .leading, spacing: 3) {
+                    Text(room.name)
+                        .font(.system(size: 17, weight: .medium))
+                        .minimumScaleFactor(0.2)
+                        .lineLimit(1)
+                    
+                    GeometryReader { geo in
+                        HStack(spacing: 6) {
+                            Text(isOpen ? "Open" : "Closed")
```

Comment:
Location/display of this seems inconsistent with the rest of the app, in my opinion.